packages/ai-jsx/src/core/memoize.tsx

```diff
@@ -109,11 +112,12 @@ export function memo(renderable: Renderable): Node {
           return finalResult;
         } else if (nextPromise == null) {
           nextPromise = generator.next().then((result) => {
+            const memoized = result.value === AI.AppendOnlyStream ? result.value : memo(result.value);
```

Comment:
Is this because it would be incorrect to memoize an append only stream, or because it's unnecessary and thus we should save the bloat from the DebugTree / rest of the system?

Reply:
Just that `memo` takes a `Renderable`, and the symbol isn't itself a `Renderable`.